<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit row</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="OGRNLabel">
     <property name="text">
      <string>OGRN</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="OGRNLineEdit">
     <property name="placeholderText">
      <string>XXXXXXXXXXXXX</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="addressLineEdit">
     <property name="placeholderText">
      <string>city, street, house, office</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameOwnerLabel">
     <property name="text">
      <string>Owner name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameOwnerLineEdit">
     <property name="placeholderText">
      <string>Last name First name Middle name(optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="phoneLabel">
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="phoneLineEdit">
     <property name="placeholderText">
      <string>+X(XXX)XXX-XX-XX</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="countProductLabel">
     <property name="text">
      <string>Product count</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="countProductLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="postCountLabel">
     <property name="text">
      <string>Post count</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="countPostLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="PriceLabel">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="priceLineEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
